src/partials/body-landing-page-capella.hbs

```diff
@@ -13,6 +13,9 @@
 <main class="capella-article" data-ceiling="topbar">
 <article class="doc landing-page-capella">
 {{{page.contents}}}
+<div class="ending-image">
+    <img src="./_images/lineplus.svg" />
+</div>
```

Comment:
This image is not displaying at all on the Capella intro page by adding it in here. We need to troubleshoot.

Reply:
Ah ok, it's a path issue.
We can move it into the UI root, and then it won't have that problem.

In the meantime it displays on https://preview.docs-test.couchbase.com/hakim-capella-landing/cloud/index.html 
(needs some minor CSS tweaking outdent it slightly)